src/style.css

```
@@ -0,0 +1,26 @@
+body,
+div {
+  margin: 0;
+}
```

`div` hasn`t margins
```suggestion
body {
  margin: 0;
}
```